INSTALL.md

```diff
@@ -1,19 +1,24 @@
-# Install
+# Installation Guides
 
-Here is a video walkthought going thought the install process :
+In order to install and use Pipelines-as-Code, you need to 
+* Install the Pipeline-as-Code infrastructure on your cluster 
```

Comment:
so Pipeline with s or without s ? I am confuse :)

The project and this org is with a S

it seems we call it "Openshift Pipelines with a S everywhere too"

Reply:
Updated